src/plugins/imagery/components/ImageryView.vue

```diff
@@ -653,7 +707,7 @@ export default {
             window.clearInterval(this.durationTracker);
         },
         updateDuration() {
-            let currentTime = this.openmct.time.clock() && this.openmct.time.clock().currentValue();
+            let currentTime = this.timeContext.clock() && this.timeContext.clock().currentValue();
             this.numericDuration = currentTime - this.parsedSelectedTime;
```

Comment:
when `this.timeContext.clock()` is null then `currentTime` ll be `undefined` and ll cause `this.numericDuration` to be `NaN`

Reply:
Ok I've updated this.